src/lineup/internal/panel/PanelTabContainer.ts

```diff
@@ -0,0 +1,114 @@
+import {PanelTab} from './PanelTab';
+import {PanelNavButton} from './PanelButton';
+
+
+
+/**
+ * The header of the PanelTab
+ * Contains the PanelNavButtons that toggle the PanelTab
+ */
+class PanelTabHeader {
+  public node: HTMLElement;
+  public navButtons: PanelNavButton[] = [];
```

Comment:
the PanelNavButtons are never pushed to this array, so it'll always be emtpy. if you push the buttons to this array, please make it private and readonly

Reply:
Thanks. We forgot to remove this in the last refactoring. I removed the variable now.